src/main/java/org/folio/service/consortium/ConsortiumConfigurationService.java

```diff
@@ -63,12 +63,12 @@ private CompletableFuture<Optional<ConsortiumConfiguration>> getConsortiumConfig
       .map(jsonObject -> jsonObject.getJsonArray(USER_TENANTS_ARRAY_IDENTIFIER))
       .map(userTenants -> {
         if (userTenants.isEmpty()) {
-          logger.debug("Central tenant and consortium id not found");
+          logger.info("getConsortiumConfigurationFromRemote:: Central tenant and consortium id not found");
```

Comment:
for non ecs setup this log would be everywhere and support during investigating of RRT issues can think that something bad is happening by reading this message

Reply:
> for non ecs setup this log would be everywhere and support during investigating of RRT issues can think that something bad is happening by reading this message

I will change it back to debug mode